ftplugin/satysfi.vim

```diff
@@ -0,0 +1 @@
+set matchpairs+=<:>
```

Comment:
Could you modify this to use `setlocal`? I'm not an expert of vim but [there's an example which prefers `setlocal` over `set`](https://github.com/rust-lang/rust.vim/blob/12f549f9e4939bca53c8a87e6921a36fb143af9a/ftplugin/rust.vim#L192).

```suggestion
setlocal matchpairs+=<:>
```